libs/model-assessment/src/lib/ModelAssessmentDashboard/Cohort/ProcessPreBuiltCohort.ts

```diff
@@ -118,14 +118,9 @@ function translatePreBuiltCohortFilterForTarget(
     jointDataset.getModelType() === ModelTypes.Binary
   ) {
     const modelClasses = jointDataset.getModelClasses();
-    const index: number[] = [];
-    for (const modelClass of preBuiltCohortFilter.arg) {
-      const indexModelClass = modelClasses.indexOf(modelClass);
-
-      if (indexModelClass !== -1) {
-        index.push(indexModelClass);
-      }
-    }
+    const index = preBuiltCohortFilter.arg
+      .map((modelClass) => modelClasses.indexOf(modelClass))
```

Comment:
might using the reduce function be even better?  
https://stackoverflow.com/questions/16037049/why-does-javascript-map-function-return-undefined

like this example they had:
```
var arr = ['a','b',1];
var results = arr.reduce((results, item) => {
    if (typeof item === 'string') results.push(modify(item)) // modify is a fictitious function that would apply some change to the items in the array
    return results
}, [])
```
map and filter also looks fine to me though